test/acceptance/list_test.exs

```diff
@@ -132,6 +132,92 @@ defmodule Acceptance.ListTest do
       assert as_html(markdown) == {:ok, html, messages}
     end
 
+    test "tight lists have no paragraph wrappers" do
+      markdown = """
+      - a
+      - b
+      """
+      html = """
+      <ul>
+      <li>a
+      </li>
+      <li>b
+      </li>
+      </ul>
+      """
+      messages = []
+
+      assert as_html(markdown) == {:ok, html, messages}
+    end
+
+    test "loose lists have paragraph wrappers" do
+      # Here the blank line makes this a loose list
+      markdown = """
+      - a
+
+      - b
+      """
```

Comment:
I noticed these strings are collapsed in most places with newline literals.  I find this easier to read, but I'm happy to collapse them to match the others once the other changes are ready.

Reply:
Never mind I will collapse them eventually, or unify the acceptance tests the other way round.